src/prefect/utilities/callables.py

```diff
@@ -149,10 +149,18 @@ class ModelConfig:
         else:
             name = param.name
 
+        # If an override is provided, use that, otherwise use the default value
+        # from the flow definition
+        parameter_default = None
+        if overrides.get(param.name) is not None:
```

Comment:
We should use `if param.name in overrides` instead — otherwise we're not letting you override with a value of `None`